### PR TITLE
Private pass2

### PR DESCRIPTION
**Pull request by PabstMirror**
Mostly privates.

Functional change:
* frag/doSpall - renamed _foundObjects 
* removed common,subString (nothing used it and select is better)

### CODE REVIEW

File: addons/common/functions/fnc_exportConfig.sqf [hunk 1]
```diff
@@ -9,7 +9,7 @@
 private "_fnc_logEntries";
 
 _fnc_logEntries = {
-  private ["_c", "_d", "_p", "_t", "_e"];
+  private ["_c", "_d", "_p", "_t", "_e", "_a", "_i"]
```
Comment (KoffeinFlummi) [inline addons/common/functions/fnc_exportConfig.sqf]: missing ;?

File: addons/frag/functions/fnc_doSpall.sqf [hunk 2]
```diff
@@ -4,11 +4,7 @@
     GVAR(traceFrags) = true;
 #endif
 // ACE_player sideChat "WAAAAAAAAAAAAAAAAAAAAA";    
-private ["_params", "_initialData", "_hpData", "_roundType", "_round", "_object", "_caliber", "_explosive",
-    "_idh", "_alive", "_exit", "_vm", "_velocity", "_unitDir", "_oldVelocity", "_curVelocity", "_diff", "_polar",
-    "_pos", "_spallPos", "_i", "_pos1", "_pos2", "_blah", "_data", "_spallPolar", "_c", "_warn", "_m", "_k",
-    "_gC", "_shellType", "_fragPower", "_spread", "_spallCount", "_elev", "_dir", "_vel", "_spallFragVect",
-    "_fragment", "_index", "_hitData", "_fragTypes", "_fragType", "_foundObjects"];
+private ["_params", "_initialData", "_hpData", "_roundType", "_round", "_object", "_caliber", "_explosive", "_idh", "_alive", "_exit", "_vm", "_velocity", "_unitDir", "_oldVelocity", "_curVelocity", "_diff", "_polar", "_pos", "_spallPos", "_i", "_pos1", "_pos2", "_blah", "_data", "_spallPolar", "_c", "_warn", "_m", "_k", "_gC", "_shellType", "_fragPower", "_spread", "_spallCount", "_elev", "_dir", "_vel", "_spallFragVect", "_fragment", "_index", "_hitData", "_fragTypes", "_fragType", "_foundObjects"];
```
Comment (KoffeinFlummi) [inline addons/frag/functions/fnc_doSpall.sqf]: oh god.

Comment (jonpas) [inline addons/frag/functions/fnc_doSpall.sqf]: :banana:'s fault

**Comment by PabstMirror** [comment]
Closing this to break down to individual modules, to make merge conflicts less afwul.

**Pull request closed without merging**